docs/pages/reference/config.mdx

```diff
@@ -867,3 +867,10 @@ Replaced `teleport.auth_servers` with either `teleport.auth_server` or `teleport
 **Introduced in Teleport v8**
 
 Added support for multiplexing via `auth_service.proxy_listener_mode`.
+
+Removed the default values being set for:
```

Comment:
Since this changelog is only going to grow, I would be in favor of keeping all changelog entries in the main changelog page, `docs/pages/changelog.mdx`. Otherwise, there's not really a reason to expect a changelog here, and users will need to remember that there's both a main changelog and a config-specific changelog.

Reply:
I've removed it from this page